bin/logger

```diff
@@ -135,25 +135,36 @@ def _write(s):
 
 
 def _parse_header(data):
-    return dict([x.split(':') for x in data.split()])
+    return dict([x.split(":") for x in data.split()])
+
+
+# Patterns in log lines that identify them as not being errors, even if the
+# log line was written to stderr.
+NON_ERROR_MARKERS = ["[gunicorn.error:INFO]", "Starting monitor"]
 
 
 def _log_payload(payload, err=False):
-    headerdata, data = payload.split('\n', 1)
+    headerdata, data = payload.split("\n", 1)
     header = _parse_header(headerdata)
-    name = header['processname']
+    name = header["processname"]
     if err:
-        name += ' (stderr)'
-    prefix = '{name:{width}} | '.format(name=name, width=WIDTH)
+        name += " (stderr)"
+    prefix = "{name:{width}} | ".format(name=name, width=WIDTH)
+
     for line in data.splitlines():
-        output_line = prefix + line + '\n'
+        format_as_error = err
+        for marker in NON_ERROR_MARKERS:
+            format_as_error = format_as_error and (marker not in line)
```

Comment:
I think this is probably a good use-case for Python's `any()` or `all()` builtin